### PR TITLE
SPRT-147: Fixing issues

### PR DESCRIPTION
**Pull request by PabloBerrettoni**
Changes based on the issue https://pukarait.atlassian.net/browse/SPRT-147
Removed an empty slide that appeared in the carousel.
Made it so the default image in the carousel is the same as in the miniature of the site.

### CODE REVIEW

File: pages/places/_id/place-details.vue [hunk 1]
```diff
@@ -17,12 +16,15 @@
...
       <i class="fa-solid fa-circle-left rounded-full text-gray-700 md:active:text-gray-500 text-4xl transform translate-x-2" @click="prevBtn" />
-      <a v-for="image, index in wikiImages" :href="image" :class="['w-11/12', 'overflow-hidden', 'md:w-5/6', 'transition', 'rounded-lg', 'shadow-md',{ 'hidden': active !== index }]">
+      <a :href="uriImg[0]" :class="['flex', 'w-11/12', 'md:w-5/6', 'transition', 'rounded-lg', 'shadow-md', 'overflow-hidden', { 'hidden': active !== 0 }]">
+        <img :key="'images'+ 0" :ref="'image-n-'+ 0" :src="uriImg[0]" :class="['w-full', 'md:p-0', 'h-64', 'md:h-96', 'object-cover', { 'hidden': active !== 0 }]">
+      </a>
+      <a v-for="image, index in wikiImages" :key="'link'+ (index + 1)" :href="image" :class="['w-11/12', 'overflow-hidden', 'md:w-5/6', 'transition', 'rounded-lg', 'shadow-md',{ 'hidden': active !== index+1 }]">
         <img
-          :key="'images'+index"
-          :ref="'image-n-'+index"
+          :key="'images'+ (index + 1)"
```
Comment (njrzr) [inline pages/places/_id/place-details.vue:24]: @PabloBerrettoni why is this index + 1? Can you explain it.

File: pages/places/_id/place-details.vue [hunk 2]
```diff
@@ -218,13 +222,13 @@ export default {
 
     prevBtn () {
       if (this.active < 1) {
-        this.active = (Object.keys(this.$refs).length - 1)
+        this.active = (Object.keys(this.$refs).length - 3)
```
Comment (njrzr) [inline pages/places/_id/place-details.vue:225]: @PabloBerrettoni something else was added?

**Review by njrzr — APPROVED** [review]

File: pages/places/_id/place-details.vue (hunk 1 above)
Comment (PabloBerrettoni) [reply pages/places/_id/place-details.vue:24]: this is in order to account for the inclusion of the default miniature in the first place of the carousel.

File: pages/places/_id/place-details.vue (hunk 2 above)
Comment (PabloBerrettoni) [reply pages/places/_id/place-details.vue:225]: Yes, the ref from the default miniature image in the carousel.

**Pull request merged**